android/src/main/java/io/bluedot/BluedotPointSdkModule.java

```diff
@@ -353,9 +355,21 @@ public void setCustomEventMetaData(ReadableMap metaData) {
     }
 
     @ReactMethod
-    public void setNotificationIDResourceID(String resourceID) {
-        // the setNotificationIDResourceID method is added to keep consistency with the
-        // iOS implementation
+    public void setNotificationIDResourceID(String resourceName) {
+        // find the resourceID int from the resourceIDString passed in
+        String packageName = reactContext.getPackageName();
+        int resourceID = reactContext.getResources().getIdentifier(resourceName, "drawable", packageName);
+        if (resourceID == 0) {
+            // not found in drawble, try mipmap
+            resourceID = reactContext.getResources().getIdentifier(resourceName, "mipmap", packageName);
```

Comment:
Should probably try mipmap first then drawable - it's possible to have both and mipmap is better.